File: YoonJeong/011주차/숫자문자열과영단어.java
import java.lang.NumberFormatException;

class Solution {
    public int solution(String s) {
        String[] alphas = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        
        for (int i = 0; i < 10; i++) {
            s = s.replaceAll(alphas[i], Integer.toString(i));
        }
        
        return Integer.parseInt(s);
    }
}

File: YoonJeong/011주차/숫자문자열과영단어.kt
class Solution {
    fun solution(s: String): Int {
        var ss = s
        var alpha: Array<String> = arrayOf("zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine")

        for (i in 0..9) {
            ss = ss.replace(alpha[i], "${i}")
        }

        return ss.toInt()
    }
}
